### ISSUE TITLE
allow specifying a polynomial basis 

### ISSUE THREAD

**Issue by lkapelevich** [lanl-ansi/MomentOpt.jl#7]
Are there plans to allow specifying an arbitrary basis for measures in here? Is this what https://github.com/lanl-ansi/MomentOpt.jl/blob/master/src/meas.jl#L10 is a placeholder for?

**Comment by tweisser** [comment]
Yes, indeed. There is going to be a basis for the moment matrix and comparing coefficients.
